core_backend/app/question_answer/routers.py

```diff
@@ -230,14 +235,12 @@ async def get_semantic_matches(
     """
     if not isinstance(response, QueryResponseError):
         metadata = create_langfuse_metadata(query_id=response.query_id, user_id=user_id)
-        content_response = convert_search_results_to_schema(
```

Comment:
now get_similar_content_async directly returns the schema object, so this function is not necessary